Mailer/app/models/Repositories/UserSubscriptionsRepository.php

```diff
@@ -149,4 +159,13 @@ public function getAllSubscribersDataForMailTypes(array $mailTypeIds)
             ->where('mail_type_id', $mailTypeIds)
             ->group('mail_type_id, subscribed');
     }
+
+    public function unsubscribeUserVariant(ActiveRow $mailType, $userId, $variantId, $email, $utmParams = [], $emitEvent = true)
```

Comment:
Please swap `$variantId` and `$email`. Logically `userId` and `email` should be next to each other as they represent the same kind of information (user identification)